src/car.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const { CarBlockIterator } = require('@ipld/car')
+const { Block } = require('multiformats/block')
+const raw = require('multiformats/codecs/raw')
+const cbor = require('@ipld/dag-cbor')
+const pb = require('@ipld/dag-pb')
+
+const MAX_BLOCK_SIZE = 1 << 20 // Maximum permitted block size in bytes (1MiB).
```

Comment:
```suggestion
const MAX_BLOCK_SIZE = 1 << 21 // Maximum permitted block size in bytes (2MiB).
```
since https://github.com/web3-storage/web3.storage/pull/1269/files